website/addons/dataverse/client.py

```diff
@@ -29,7 +29,10 @@ def connect_from_settings(node_settings):
 
 def connect_or_401(host, token):
```

Comment:
This should be changed to `connect_or_error`, since it now doesn't just return a 401 in the event of an error. 

I can make this change when I merge.